test/site/www/table.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>two</title>
+    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1">
+    <link rel="stylesheet" href="css/bootstrap.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+	<table>
+	    <tr><td>2015-03-01</td><td>12:00</td></tr>
+	    <tr><td>2015-03-02</td><td>12:00</td></tr>
+	</table>
```

Comment:
can we move that table to `index.html`? I don't thing we need another site only for these two columns